car.rb

```diff
@@ -0,0 +1,16 @@
+class Car < Transport
+
+  attr_accessor :registration_number
+  attr_reader   :speed , :max_weight 
+
+  CAR_MAX_WEIGHT = 100
+
+  def initialize(current_weight , registration_number, is_available)  	
+  	@max_weight = CAR_MAX_WEIGHT
```

Comment:
u can use `rubocop` gem for taking care of style guide

Reply:
started using rubocop